packages/generator/README.md

```diff
@@ -135,6 +108,39 @@ Autocomplete will also show the available properties of the data returned by SDK
 
 ![Result Autocomplete](https://github.com/SalesforceCommerceCloud/commerce-sdk/raw/master/packages/generator/images/ResultAutocomplete.jpg?raw=true "Result Autocomplete")
 
+## Caching
+
+The SDK currently supports two types of caches - In-memory and Redis. Both the implementations respect [standard cache headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control). If you would like to use another type of cache, you can write your own implementation of the [CacheManager](../core/src/base/cacheManager.ts)
+
+### In-memory cache
+In-memory caching of responses is enabled by default. To disable caching for a client, set cacheManager to 'null'.
+```javascript
+const config = {
+    cacheManager: null,
+    parameters: {
+        clientId: "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+        organizationId: "f_ecom_bblx_stg",
+        shortCode: "0dnz6oep",
+        siteId: "RefArch"
+    }
+}
+```
+
+### Redis cache
+To use a Redis cache, instantiate a CacheManagerRedis object with your Redis URL and put it in your client config object.
+```javascript
+const cacheManager = new CacheManagerRedis({ connection: "redis://localhost:6379" });
+const config = {
+    cacheManager,
```

Comment:
Is this the preferred way?

Reply:
Oh I guess that works as a shorthand since the name is the same, right? I think for the sake of the doc it makes it a little clearer so thanks for switching it.